ha/core/system_health/system_health.py

```diff
@@ -85,6 +85,14 @@ def get_node_status(self, node_id, **kwargs):
         """
         get node status method. This method is for returning a status of a node.
         """
+        try:
+            # Prepare key and read the health value.
+            key = self._prepare_key(component='node', node_id=node_id, **kwargs)
```

Comment:
@akash2144 As discussed we will get only the node_id (And may be some additional filters in future) in this API call. Already we have a mechanism to store the node_id to cluster/site/rack/storageset mapping in the store. So, lets read this mapping and use it here. Need to update the code to store the same as well.

cc @ajaysrivas

Reply:
Done, values read from mapping.